src/app/resourcebundles/data/consumption/as.properties

```diff
@@ -1930,4 +1930,5 @@ frmelmnts.lbl.confirmEvidenceDelete = Are you sure you want to delete the attach
 frmelmnts.lbl.sectionName = Section name
 frmelmnts.lbl.enterResponse = Enter your response
 frmelmnts.lbl.lastModifiedOn = Last modified on
-frmelmnts.lbl.enableDebugMode = ডিবাগ মোড সক্ষম কৰক
\ No newline at end of file
+frmelmnts.lbl.enableDebugMode = ডিবাগ মোড সক্ষম কৰক
```

Comment:
@mithun30052001 please fix the whitespace

Reply:
Sure prince